internal/c/start.go

```diff
@@ -8,6 +8,27 @@ import (
 	"time"
 )
 
+// capatazKey is an internal type for the capataz keys
+type capatazKey string
+
+// workerNameKey is an internal representation of the worker name in the
+// worker context. If you reverse engineer, you are on your own.
+//
+var workerNameKey capatazKey = "__capataz.worker.runtime_name__"
```

Comment:
What does this type alias do? I don't see it used anywhere.

Reply:
because context value has an `interface{}` value as a key, the linter forces us to create a custom type so that other people cannot pluck the values (if they don't have a value with the key type, they cannot remove the value at all)